components/ChallengeMaterial.tsx

```diff
@@ -102,6 +104,47 @@ export const ChallengeQuestionCard: React.FC<ChallengeQuestionCardProps> = ({
   const { data } = React.useContext(SessionContext)
   const username = _.get(data, 'userInfo.username', '')
   const diff = _.get(currentChallenge, 'submission.diff', false)
+  let files = null
+
+  if (diff) files = gitDiffParser.parse(diff)
+
+  const renderFile = ({ hunks, newPath }: File) => {
+    const oldValue: String[] = []
+    const newValue: String[] = []
+
+    hunks.forEach(hunk => {
+      hunk.changes.forEach(change => {
+        if (change.isDelete) oldValue.push(change.content)
+        else if (change.isInsert) newValue.push(change.content)
+        else {
+          oldValue.push(change.content)
+          newValue.push(change.content)
+        }
+      })
+    })
+
+    const syntaxHighlight = (str: string): any => {
```

Comment:
Try setting return type to JSX.Element

Reply:
It won't allow me because of my first return statement when `string` doesn't exist. Also, I tried `JSX.Element | null` and it conflicted with the prop that I am passing this function into